File: Haber/java haber/Bildirim.java
public class Bildirim implements Iletisim {
    @Override
    public void bilgiGonder(Kullanici kullanici, String mesaj){
        this.bildirimGonder(kullanici.GetIp(), mesaj);
    }

    public void bildirimGonder( int ip, String mesaj){
        System.out.println(ip + " adresine " + mesaj + " gonderilmistir.");
    }
}

File: Haber/csharp haber/Bildirim.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_haber
{
    public class Bildirim : Iletisim
    {
        public void BilgiGonder(Kullanici kullanici, string mesaj){
            this.BildirimGonder(kullanici.GetIp(), mesaj);
        }

        public void BildirimGonder(int ip, string mesaj){
            Console.WriteLine(ip + " adresine " + mesaj + " gonderildi.");
        }
    }
}